<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsQU</class>
 <widget class="QWidget" name="QtWidgetsQU">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件加载</string>
  </property>
  <widget class="QLabel" name="labelWind">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地点输入</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBegin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>起点</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDone">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEnd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>终点</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxBegin">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxEnd">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>151</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelHits">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>264</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonDone</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsQU</receiver>
   <slot>Done()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonQuit</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsQU</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Done()</slot>
 </slots>
</ui>
